content/en/docs/tutorials/services/connect-applications-service.md

```diff
@@ -292,6 +292,10 @@ And also the configmap:
 ```shell
 kubectl create configmap nginxconfigmap --from-file=default.conf
 ```
+
+Note: The `default.conf` file is located in
+[https://github.com/kubernetes/examples/tree/master/staging/https-nginx/](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/).
```

Comment:
Try https://github.com/kubernetes/examples/tree/bc9ca4ca32bb28762ef216386934bef20f1f9930/staging/https-nginx/ (pinning to a commit)